app/admin/dashboard.rb

```diff
@@ -163,35 +287,24 @@
       end # tab
 
       tab :metrics do
+        # export .xlsx button
+        form action: export_metrics_path(), method: :get, style: 'text-align: right' do |f|
+          f.input :submit, :type => :submit, :value => 'Export as .xlsx', style: 'margin-bottom: 1rem'
```

Comment:
oh, and let's get rid of these "rockets", just use `type: :submit`, etc.

Reply:
🙆‍♂️ haha thank you!